### ISSUE TITLE
Feature Request: `input-id`

### ISSUE THREAD

**Issue by TheSeg** [Vuepic/vue-datepicker#557]
For accessibility reasons, there should be a `input-id` prop to define the `<input>`'s ID attribute. This allows the standard `<label id="idOfInput">` to work as intended. This matches the convention of similar props including `input-class-name`.

Consider this example:

```html
<label for="inputDate">Input Date:</label>
<input id="inputDate" type="text">
```

This request is to make a `input-id` prop and forgo using the slots method. This simplifies the code to:

```vue
<template>
    <label for="inputDate">Input Date:</label>
    <VueDatePicker v-model="date" input-id="inputDate" />
</template>
```

Currently with vue-datepicker, the only way to maintain accessibility standards is by [using slots](https://vue3datepicker.com/slots/trigger-and-input/#dp-input). While this is doable, it further complicates attempting to be accessible while using vue-datepicker. You break loosing the other properties with vue-datepicker and have to implement the entire `<input>` functionally by hand.

```vue
<template>
    <label for="inputDate">Input Date:</label>
    <VueDatePicker v-model="date">
        <template #dp-input="{ value, onInput, onEnter, onTab, onClear, onBlur, onKeypress, onPaste, isMenuOpen }">
          <input id="inputDate" type="text" :value="value" />
        </template>
    </VueDatePicker>
</template>
```

**Comment by Jasenkoo** [comment]
There is a prop called [`uid`](https://vue3datepicker.com/props/general-configuration/#uid) for setting the `id` to the input field.

**Comment by SokolovskyAlex** [comment]
Hi, @Jasenkoo !
I have the same issue
in case with **uid** result will be like this

```
<label for="**some-id**">Input Date:</label>
    <VueDatePicker v-model="date">
        <template #dp-input="{ value, onInput, onEnter, onTab, onClear, onBlur, onKeypress, onPaste, isMenuOpen }">
          <input id="dp-input-**some-id**" type="text" :value="value" />
        </template>
    </VueDatePicker>
```
![image](https://github.com/Vuepic/vue-datepicker/assets/98600404/0246514e-18c7-40af-97a0-f2d99605c1e0)

if i'm doing something wrong please tell me what

**Comment by Jasenkoo** [comment]
@SokolovskyAlex The `id` will have prefix `dp-input-` so when you pass the `uid="my-id"`, the input `id` will be `dp-input-my-id`
